css/default/_stars-list.scss

```diff
@@ -0,0 +1,59 @@
+      }
+      @include bem('stars-list', 'star', 'empty') {
+        background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACYAAAAmCAYAAACoPemuAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAB1WlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOkNvbXByZXNzaW9uPjE8L3RpZmY6Q29tcHJlc3Npb24+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgICAgIDx0aWZmOlBob3RvbWV0cmljSW50ZXJwcmV0YXRpb24+MjwvdGlmZjpQaG90b21ldHJpY0ludGVycHJldGF0aW9uPgogICAgICA8L3JkZjpEZXNjcmlwdGlvbj4KICAgPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KAtiABQAABJ5JREFUWAndWEtIVFEYnpfi4At1fBNlIxGlqYsRN0ELwcwwQyQX4aJFYIseii1cBLbQQJR2ikILaWGYlNAm3Gc5RRAWmIql+GAc0/AxgzpO/3e5v5yu13vncanowOGcuec/3/+d7/zn3P+OyfS/l+bmZheqUeu0GQVktVqvy1huIzAtRoBAqWAweA7VKNUMISaoZRL70Sw6amKsVkxMzE9Uo1SLmhgrVFJSEkCFSvzsrykmqlVVVXUF1SjVolKMlYFSdrvdimqUahET6+zszEM8QaHKyspLvG3os2qw4efhthETW1lZuQtnUCg+Pj6OHaPPqrENj4XTRkSM1YIjUS12bIRqYROrq6uzsxIFBQVeUS0mplQNc3gs1NasZgig0tLSrOXlZQeN51ssluNmsznbZrMd393dTcYcxFFra2uFGjGMb21t+dvb21+L9nt7e98pLpf29/e/k8l0VlaWd3x8fHloaMiHOWIxy68QVeeiIfeJoD8nJ2ejoaHBlZKScoKfq7Vra2vfBgYG3IuLi4lE6CAOlbZYpJK0RIyO/UNxIpwnJCT4HA7HXnp6upVWFk9kHNnZ2Rl0JRxTAofy2+fzzS8tLXmIpJd2YovCIeD1em2bm5t2pe9AIPBA2kqoxuTy8vJWGhsb60JxZpRNT0/P0OzsbDoEAamuri63FeBjY2OLZWVlXymWzpP8ydPT026Xy3XWKMdaOGqkYC8RQwfkampq3mxvb1f8KXIiKQqZW21tbZ/BBeWAGH6Mjo6ug9zOzs6F1dXVVChXXFx8hrZZ9fRiTiSFTmqwr6/vObYPgZ+Wlna7paVlVsT6jRgGQK66uvodk5ucnHxfVFR0mgDCvvNER9ynQxDo7e0dnpubk0jRyb6nJAXbQ8TwUCS3vr6eOjU19aGwsNAZGxsbVSqOu62/v39kYWFBk9SRxNTI0bZ+pAMS1YGgmHoRCin419weSEx32Q3EAQDpHvqCSZEUzGVSwFTbPhFXkxgMm5qaflCwvkV/ZmZmHm0khecCC5h6GLrEAID3JFrc/mgjKTyXsfQwQiJGIKcAlJmZGXGMCXMlrKiJdXd3p+Jdhjg7KpPQc4JxzAUGsICpN0dXMQpYJ0AyMjJ2tMBwaaJq2TAGY2rZhnIv5QMgNzdX9c5DajM4OOjGLQ47JAH19fWqKREwiBTMgKn5V4IuMSSJQELqg5YLLsvh4eFXExMTOBASKYyBYEdHxzfKbt/X1tZeFrefMRiTsdRaXWI4RRQXJqfTKeVheKVQxvlCJsSn9DXlan1wQPfVTWoqME71LdJvyoiv4tNOxvCEcjJ1iZET6RQlJSWdHBkZeUmpsJli6YAQxc0zxWX5iD5Wnnk8nmtMkN63ryhVD5aXl1+kZx7GpPbIopk14GuIHDzBaYqLi9vf2NhIARKt+BOlKY8VhA45wXx8uJDi5zCYmJi45vf7LbSwZFqQ5u2vqRg+RigeTACiKhGiDPMpMsxDLFQeyMTvyBnydVqYRBCm8ofOb6mOCKFJjAzzYQyFwiEkOkBfXoibCcoKAvvIBeoRM9Gn1v1QFVISUv4WCdKYtGilzT//+xfFkXzR1QryOQAAAABJRU5ErkJggg==");
```

Comment:
> Anyway, as we are using scss and commiting the png files, we should use a mixin that does the base64 stuff for us.

Yes, I didn't find any "simple" mixin (I mean, without embedding the whole Compass/Bourbon; any idea?)